### ISSUE TITLE
Logo Design

### ISSUE THREAD

**Issue by batarian71** [metacloud/molecule#1356]
Hi. I am a graphic designer. I would like to volunteer to design a logo for this repository that you can use in the readme file. What do you say?

**Comment by retr0h** [comment]
Hi @batarian71 Molecule currently has a [logo](http://molecule.readthedocs.io/en/latest/).  However, I appreciate your offer.

**Comment by retr0h** [comment]
Hi @batarian71 it's very cool, but we already have a logo.  Thank you.

**Comment by retr0h** [comment]
@batarian71 I tell you what.  If you provide a few diff design options maybe we can use yours.

**Comment by batarian71** [comment]
Yes, this project is under creativecommons license. I do not quite understand what you want. a new logo you want?

**Comment by retr0h** [comment]
> Yes, this project is under creativecommons license. I do not quite understand what you want. a new logo you want?

A few diff style logos to choose from.  I'm not guaranteeing I would use it, but curious what you could provide.

**Comment by retr0h** [comment]
Maybe a little more pixelated and futuristic ?
